src/pymoca/backends/casadi/model.py

```diff
@@ -50,9 +50,11 @@ def from_dict(cls, d):
         variable.aliases = d['aliases']
         return variable
 
-
-DelayedState = namedtuple('DelayedState', ['name', 'origin', 'delay'])
-
+class DelayedState:
+    def __init__(self, name, origin, delay):
+        self.name = name
+        self.origin = origin
+        self.delay = delay
```

Comment:
Why this?

Reply:
Have to overwrite field. Not possible with tuple.